src/sql/parts/objectExplorer/common/objectExplorerService.ts

```diff
@@ -77,6 +77,8 @@ export interface IObjectExplorerService {
 	getTreeNode(connectionId: string, nodePath: string): Thenable<TreeNode>;
 
 	refreshNodeInView(connectionId: string, nodePath: string): Thenable<TreeNode>;
+
+	getNodeActions(connectionId: string, nodePath: string): Thenable<string[]>;
```

Comment:
Is this only added so that it can be tested? We should find another way to test it if so

Reply:
can you at least add a tsdoc comment that this is just for testing so that we know it doesn't need to be implemented for anything else that implements this interface? probably good to make it optional too